golem_core/mid/simple_scorer.py

```diff
@@ -81,8 +87,11 @@ async def _wait_until_ready(self) -> None:
         start = datetime.now()
 
         while True:
+            await asyncio.sleep(0.1)
```

Comment:
Moving sleep at the beginning looks like not a good idea for me. It introduces unwanted delay, if work is node sooner.

Reply:
`sleep` has to be before `continue` block otherwise it will deadlock all other asyncio tasks.